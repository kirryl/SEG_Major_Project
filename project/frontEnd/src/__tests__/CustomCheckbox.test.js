import renderer from 'react-test-renderer';
import Checkbox from '../components/CustomCheckbox/SimpleCheckbox';
import React from 'react';
import { shallow } from 'enzyme';
import { mount } from 'enzyme';
import enzymeConfig from '../../enzymeConfig'

describe('<Checkbox />', function(){
  let props;
  let mountedCheckbox;

  beforeEach(() => {
    props = {
      simple: undefined,
      round: undefined,
      icon: undefined,
      title: undefined,
      description: undefined,
    };
    mountedCheckbox = undefined;
  });
    it('renders correctly', function(){
        const component = renderer.create(<Checkbox />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

//     it('Should capture checkbox ticked correctly onChange', function(){
//
// })
})
