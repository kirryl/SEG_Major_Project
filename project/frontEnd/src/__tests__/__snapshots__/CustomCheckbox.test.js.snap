// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> renders correctly 1`] = `
<div
  className="position-relative form-check"
>
  <label
    className="form-check-label"
  >
    <input
      className="form-check-input"
      type="checkbox"
    />
    <span
      className="form-check-sign"
    />
    
  </label>
</div>
`;

exports[`<Checkbox /> should render correctly 1`] = `
<div
  className="position-relative form-check"
>
  <label
    className="form-check-label"
  >
    <input
      className="form-check-input"
      type="checkbox"
    />
    <span
      className="form-check-sign"
    />
    
  </label>
</div>
`;
